modules/KatzCentrality/README.md

```diff
@@ -14,20 +14,23 @@ The Katz (sometimes also called Katz-Bonacich) centrality measures with
 the parameter of the network nodes are defined by the `n x 1` vector:
 
 
-![Katz Centrality Formula](./KatzCentralityFormula.png)
+![Katz Centrality Formula](https://github.com/gephi/gephi-plugins/raw/katz-centrality-plugin/modules/KatzCentrality/KatzCentralityFormula.png)
```

Comment:
@mbastian thank you for converting the image links to GitHub I'll make sure to use GitHub in the future.

Reply:
No worries, I found this solution more sustainable than the image storage online services